sklearn/neighbors/tests/test_lof.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from sklearn import neighbors
+
+from numpy.testing import assert_array_equal
+
+from sklearn.metrics import roc_auc_score
+from sklearn.utils import check_random_state
+from sklearn.utils.testing import assert_greater
+from sklearn.utils.testing import assert_array_almost_equal
+
+
+def test_lof():
+    # toy sample (the last two samples are outliers)
+    X = [[-2, -1], [-1, -1], [-1, -2], [1, 1], [1, 2], [2, 1], [5, 3], [-4, 2]]
+
+    # Test LOF
+    clf = neighbors.LOF()
+    clf.fit(X)
+    pred = clf.predict()
+    assert_array_equal(clf._fit_X, X)
```

Comment:
Sorry I don't see your point, the ranking is received directly in `pred`...
`clf._fit_X` is just the training samples (defined in the base estimator NeighborsBase), the user doesn't see or need it...